packages/serverpod_shared/test_integration/assets/load_passwords/runner_empty_passwords.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:io';
+import 'package:serverpod_shared/serverpod_shared.dart';
+
+void main() {
+  var passwords = PasswordManager(runMode: 'development').loadPasswords(
+    './config/empty_passwords.yaml',
```

Comment:
Suggestion: create this file as part of the test.

Since the file doesn't have a complex structure and is only used for a single test we might as well create it temporarily for the specific test.

Reply:
We could but since the file location is hardcoded in this file which is disconnected from the test it self I think it is nice to keep the file checked in.